### PR TITLE
Rename katib-manager to katib-db-manager

### PR DESCRIPTION
**Pull request by richardsliu**
**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**

**Checklist:**
- [ ] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate-changed-only`
    3. `make test`

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/743)

### CODE REVIEW

File: katib/katib-controller/base/katib-db-manager-deployment.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: katib-manager
+  name: katib-db-manager
   labels:
     app: katib
     component: manager
```
Comment (johnugeorge) [inline katib/katib-controller/base/katib-db-manager-deployment.yaml]: Should we change component name also? There are couple of places

**Comment by richardsliu** [comment]
@johnugeorge Done

**Pull request merged**